webpack.config.js

```diff
@@ -31,44 +30,56 @@ if (process.env.NODE_ENV === 'production') {
   )
 }
 
-const loaders = [
-  {
-    test: /\.js?$/,
-    exclude: /(node_modules|bower_components|dist)/,
-    loader: 'babel-loader'
-  }
-]
-
 module.exports = [
   {
+    // Browser
     context: path.join(__dirname, 'lib'),
     entry: './contentful-management.js',
     output: {
       path: path.join(__dirname, 'dist'),
-      filename: `contentful-management${process.env.NODE_ENV === 'production' ? '.min' : ''}.js`,
+      filename: `contentful-management${PROD ? '.min' : ''}.js`,
       libraryTarget: 'umd',
       library: 'contentfulManagement'
     },
     module: {
-      loaders: loaders
+      loaders: [
+        {
+          test: /\.js?$/,
+          exclude: /(node_modules|bower_components|dist)/,
+          loader: 'babel-loader',
+          options: {
+            env: 'browser'
+          }
+        }
+      ]
     },
-    devtool: 'source-map',
+    devtool: PROD ? false : 'source-map',
```

Comment:
I think this will cause a crash when the user try to build his app, because of `es6-promise` source maps